### PR TITLE
Fix make target for release-staging

### PR DESCRIPTION
**Pull request by SanikaGawhane**
Fix for make target that changed because of change in `cloudbuild.yaml` location.

/assign @codenrhoden 
/assign @kkeshavamurthy

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ steps:
       - TAG=$_GIT_TAG
       - DOCKER_BUILDKIT=1
     args:
-      - release-staging
+      - -C images/capi release-staging
```
Comment (codenrhoden) [inline cloudbuild.yaml:14]: I didn't even think about putting it here. This works!

Comment (SanikaGawhane) [reply cloudbuild.yaml:14]: Thought this was a better place for the fix. Glad this works!

**Pull request merged**